connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorHeartbeatConfig.java

```diff
@@ -77,7 +79,9 @@ short heartbeatsTopicReplicationFactor() {
                     ConfigDef.Importance.LOW,
                     HEARTBEATS_TOPIC_REPLICATION_FACTOR_DOC);
 
+    protected final static ConfigDef CONNECTOR_CONFIG_DEF = new ConfigDef(mergeConnectorConfigDef(HEARTBEAT_CONFIG_DEF));
+
     public static void main(String[] args) {
-        System.out.println(CONNECTOR_CONFIG_DEF.toHtml(4, config -> "mirror_heartbeat_" + config));
+        System.out.println(HEARTBEAT_CONFIG_DEF.toHtml(4, config -> "mirror_heartbeat_" + config));
```

Comment:
With the above suggestion, this can now become:
```java
        System.out.println(defineHeartbeatConfig(new ConfigDef()).toHtml(4, config -> "mirror_heartbeat_" + config));
```